### ISSUE TITLE
Error: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by bastienrobert** [zekenie/sequelize-erd#2]
Just setup sequelize-erd with this line in my package.json ```sequelize-erd --source ./app/models/ --destination ./erd.svg``` and get this error when I try a ```npm run erd```

```
> express-mvc@1.0.0 erd /Users/bastien/Documents/WWW/express-mvc
> sequelize-erd --source ./app/models/ --destination ./erd.svg

/Users/bastien/Documents/WWW/express-mvc/node_modules/sequelize-erd/index.js:31
  const models = Object.values(db.models);
                        ^

TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at module.exports.path (/Users/bastien/Documents/WWW/express-mvc/node_modules/sequelize-erd/index.js:31:25)
    at Object.<anonymous> (/Users/bastien/Documents/WWW/express-mvc/node_modules/sequelize-erd/bin/generate:14:13)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-mvc@1.0.0 erd: `sequelize-erd --source ./app/models/ --destination ./erd.svg`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the express-mvc@1.0.0 erd script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bastien/.npm/_logs/2017-08-08T20_18_45_782Z-debug.log
```

**Comment by bastienrobert** [comment]
Oops, I forget the path :
- Models : ./app/models
- Migrations : ./db/migrations
- Config : ./config/sequelize.json

Sequelize run smoothly when I want to create a model.

**Comment by raphodn** [comment]
so what command did you do in the end ? I have the same problem

**Comment by bastienrobert** [comment]
Issue still open, didn't get any answer, I wait for one

**Comment by mikew** [comment]
If you're using ES6 exports, like `export default db`, then this package breaks. [This line](https://github.com/zekenie/sequelize-erd/blob/master/index.js#L29) needs to look for `require(path).default`, or use the package programatically.

**Comment by nadeemyasin61** [comment]
Just in case if its helpful for the future visitors, I managed to get it working with the following code. Please note that all of my models are defined `models` folder

`'use strict';

var fs        = require('fs');
var path      = require('path');
var Sequelize = require('sequelize');
var SequelizeErd = require('sequelize-erd');
var basename  = path.basename(module.filename);
var env       = process.env.NODE_ENV || 'development';
var db        = {};
var sqlz = {}

fs
  .readdirSync(path.join(__dirname,"../models/"))
  .filter(function(file) {
    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
  })
  .forEach(function(file) {
    if(file != 'index.js' ){
      var model = sequelize['import'](path.join(__dirname,"../models/", file));
      db[model.name] = model;
    }
  });

Object.keys(db).forEach(function(modelName) {
  if (db[modelName].associate) {
    db[modelName].associate(db);
  }
});

const svg = SequelizeErd(sqlz);
fs.writeFileSync('./erd.svg', svg);
`

**Comment by zekenie** [comment]
Sorry about that! This issue happens when the `source` path specified is not exporting a sequelize instance. I'm going to publish a version bump today that issues a better warning.

**Comment by zekenie** [comment]
Closed by cfbe640d16438cb5d215812e0c9077fb3feed36d
